phoneFormat.js

```diff
@@ -0,0 +1,129 @@
+jQuery(document).ready(function(){
+	(function($) {
+		if ($('#en__field_supporter_NOT_TAGGED_4').length > 0) { // check if phone form input is on page
+			var countrycode = 64;
```

Comment:
I don't know the desired outcome, but if you need to check the number (and or possibly prevent the submission) a library could be handy
Google libphone is nice for this kind of stuff.
https://github.com/google/libphonenumber

Reply:
Thanks. I have removed this and used your php suggested method instead.
 // check if phone form input is on page